modules/configuring-layer-two-switched-topology.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/configuring-secondary-networks.adoc
+
+:_content-type: CONCEPT
+[id="configuring-layer-two-switched-topology_{context}"]
+= Configuring a layer 2 switched topology
+
+The switched (_layer 2_) topology networks interconnect the workloads through a cluster-wide logical switch.
+
+[NOTE]
+====
+Layer 2 switched topology networks only allow for the transfer of data packets from server to server within a data center.
+====
+
+The following `NetworkAttachmentDefinition` custom resource definition (CRD) YAML describes the fields needed to configure a switched secondary network.
+
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: l2-network
+  namespace: ns1
+spec:
+  config: |2
+    {
+            "cniVersion": "0.3.1",
+            "name": "l2-network", <1>
+            "type": "ovn-k8s-cni-overlay", <2>
+            "topology":"layer2", <3>
+            "subnets": "10.100.200.0/24", <4>
+            "mtu": 1300, <5>
+            "netAttachDefName": "ns1/l2-network", <6>
+            "excludeSubnets": "10.100.200.0/29" <7>
+    }
+----
+<1> The name of the network. This attribute is not namespaced.
+<2> `ovn-k8s-cni-overlay` is required as the config type.
+<3> `layer2` is required as the config topology when setting a layer 2 switched topology.
+<4> A comma separate list of subnets. When multiple subnets are provided, the user gets an IP from each subnet. This attribute indicates both the subnet across the cluster, and per node. In this example, there is a `/14` subnet for the network. If this attribute is omitted, the logical switch implementing the network only provides layer 2 communication, and you must configure IPs for the pods. Port security only prevents MAC spoofing.
```

Comment:
This is wrong; for L2 there isn't a subnet per node - you just define the subnet for the entire cluster.

Reply:
Would we want to also refer this can be IPv6 / dual stack configuration ?

I do not know how the other docs are stating this, thus, I'm unsure that is implicit / obvious to the user.